pkg/apis/pipeline/v1/matrix_types_test.go

```diff
@@ -782,7 +782,7 @@ func TestPipelineTask_CountCombinations(t *testing.T) {
 	}{{
 		name: "combinations count is zero",
 		matrix: &v1.Matrix{
-			Params: v1.Params{{}}},
+			Params: v1.Params{}},
```

Comment:
The behavior is different between having no param and having a param with an empty value.

Reply:
btw, the indentation is really weird here. We should format it better.

```
&v1.Matrix{
    Params: v1.Params{},
}
// or 
&v1.Matrix{Params: v1.Params{}}
```